src/node_crypto.cc

```diff
@@ -482,12 +485,13 @@ void SecureContext::SetKey(const FunctionCallbackInfo<Value>& args) {
   if (!bio)
     return;
 
-  node::Utf8Value passphrase(env->isolate(), args[1]);
-
+  auto callback = has_password ? PasswordCallback : NoPasswordCallback;
```

Comment:
... because it really simplifies this code.

This function uses the password cb because it may have a password.

The other examples below that use `NoPasswordCallback` I agree are more clear because the usage makes it really clear that there is no password, but I think here it is unnecessarily complex, caused only because the `PasswordCallback` stopped supporting an empty password.